resources/views/movies/create.blade.php

```diff
@@ -1,7 +1,58 @@
-<?php
-/**
- * Created by PhpStorm.
- * User: joost
- * Date: 10-7-2019
- * Time: 10:33
- */
\ No newline at end of file
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                <form action="/movies" method="post">
```

Comment:
{{ url('') }} gebruiken
```suggestion
                <form action="{{ url('movies') }}" method="post">
```